docs/index.asciidoc

```diff
@@ -51,6 +52,19 @@ or in another character set other than `UTF-8`.
 
 This only affects "plain" format logs since json is `UTF-8` already.
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: plugin only sets the `message` field
+** `v1`,`v8`: Elastic Common Schema compliant behavior (`[event][original]` is also added)
+* Default value depends on which version of Logstash is running:
+** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
+** Otherwise, the default value is `disabled`
```

Comment:
You can consider deleting the default description.
default = disabled only valid until 7.last

Reply:
in general, it is not clear to user what is the default value and we didn't provide a mapping to check the version of LS and ecs. May be worth considering adding 7.last default is disabled while 8.x default is v1

Reply:
not sure if we need to re-phrase here - it's not super clear but the default value is covered:
```
  * Default value depends on which version of Logstash is running:
    ** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
    ** Otherwise, the default value is `disabled`
```
`disabled` is here for LS that does not provide `pipeline.ecs_compatibility` (the message still holds for 8.0)

@karenzone any suggestions to cause less confusion in terms of the `ecs_compatibility` defaults?
believe we purposefully did not include the "explicit" 8.0 ECS on by default here for every plugin or do we want that?

Reply:
You both make some valid points. What we have here is consistent with the text/messaging we've used for other plugins.  I recommend that we keep the text as is for now so that we can be consistent with other plugins and get this plugin published. 

To address the issue of defaults, we can come up with some messaging in a single source of truth and then reference it from all plugins.  Right now, I'm picturing a shared attribute or an include file. I will take this as an action item to come up with the best, most efficient approach to consistent messaging.